apis/python/src/tiledbsoma/__init__.py

```diff
@@ -61,15 +61,15 @@
     "TileDBPlatformConfig",
     "TileDBObject",
```

Comment:
Why are TileDBObject and TileDBArray exposed in the package?

Reply:
presumably oversight -- this is not new -- thank you for the code-review catch -- i can neaten that now or on a separate PR

Reply:
#465